00-HTML-CSS-basics/1-Basics of HTML and CSS/exercise_v.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="keywords" content="HTML,CSS">
+    <meta name="description" content="Basics of HTML and CSS">
+    <meta name="author" content="Martina Pintener">
+
+    <title>Random</title>
+
+</head>
+<body>
+    <img src="imgs/memeful.png">
+    <br>
+    <audio controls>
+        <source src="audio/ride.mp3" type="audio/mpeg" />
+        <source src="audio/ride.ogg" type="audio/ogg" />
```

Comment:
Add a text to be shown if the browser do not support any of the provided source types.